src/ng/animate.js

```diff
@@ -1,372 +1,260 @@
 'use strict';
 
 var $animateMinErr = minErr('$animate');
+var ELEMENT_NODE = 1;
+
+function mergeClasses(a,b) {
+  if (!a && !b) return '';
+  if (!a) return b;
+  if (!b) return a;
+  if (isArray(a)) a = a.join(' ');
+  if (isArray(b)) b = b.join(' ');
+  return a + ' ' + b;
+}
+
+function extractElementNode(element) {
+  for (var i = 0; i < element.length; i++) {
+    var elm = element[i];
+    if (elm.nodeType === ELEMENT_NODE) {
+      return elm;
+    }
+  }
+}
+
+var $$AnimateClassResolverProvider = [function() {
+  this.$get = [function() {
```

Comment:
If injectables (services and providers) don't have dependencies then you don't need to wrap them in an array.